crates/matrix-sdk/src/event_cache/room/mod.rs

```diff
@@ -769,72 +754,6 @@ mod private {
             Ok((output, updates_as_vector_diffs))
         }
     }
-
-    /// Create a debug string over multiple lines (one String per line),
-    /// offering a debug representation of a [`RawChunk`] loaded from disk.
-    fn raw_chunks_debug_string(mut raw_chunks: Vec<RawChunk<Event, Gap>>) -> Vec<String> {
-        let mut result = Vec::new();
-
-        // Sort the chunks by id, for the output to be deterministic.
-        raw_chunks.sort_by_key(|c| c.identifier.index());
-
-        for c in raw_chunks {
-            let content = chunk_debug_string(&c.content);
-
-            let prev =
-                c.previous.map_or_else(|| "<none>".to_owned(), |prev| prev.index().to_string());
-            let next = c.next.map_or_else(|| "<none>".to_owned(), |prev| prev.index().to_string());
-
-            let line =
-                format!("chunk #{} (prev={prev}, next={next}): {content}", c.identifier.index());
-
-            result.push(line);
-        }
-
-        result
-    }
-
-    #[cfg(test)]
-    mod tests {
-        use matrix_sdk_base::{
-            event_cache::Gap,
-            linked_chunk::{ChunkContent, ChunkIdentifier as CId, RawChunk},
-        };
-        use matrix_sdk_test::{event_factory::EventFactory, ALICE, DEFAULT_TEST_ROOM_ID};
-        use ruma::event_id;
-
-        use super::raw_chunks_debug_string;
-
-        #[test]
-        fn test_raw_chunks_debug_string() {
-            let mut raws = Vec::new();
-            let f = EventFactory::new().room(&DEFAULT_TEST_ROOM_ID).sender(*ALICE);
-
-            raws.push(RawChunk {
-                content: ChunkContent::Items(vec![
-                    f.text_msg("hey")
-                        .event_id(event_id!("$123456789101112131415617181920"))
-                        .into_event(),
-                    f.text_msg("you").event_id(event_id!("$2")).into_event(),
-                ]),
-                identifier: CId::new(1),
-                previous: Some(CId::new(0)),
-                next: None,
-            });
-
-            raws.push(RawChunk {
-                content: ChunkContent::Gap(Gap { prev_token: "prev-token".to_owned() }),
-                identifier: CId::new(0),
-                previous: None,
-                next: Some(CId::new(1)),
-            });
-
-            let output = raw_chunks_debug_string(raws);
-            assert_eq!(output.len(), 2);
-            assert_eq!(&output[0], "chunk #0 (prev=<none>, next=1): gap['prev-token']");
-            assert_eq!(&output[1], "chunk #1 (prev=0, next=<none>): events[$12345678, $2]");
-        }
-    }
```

Comment:
Can you move the test over to `RoomEvents::debug_string()`? Or transform it into a minimal test case for `chunk_debug_string` that was called internally and indirectly tested here?

Reply:
Done.